spec/models/merchant_spec.rb

```diff
@@ -48,7 +48,8 @@
       order_2.item_orders.create!(item: chain, price: chain.price, quantity: 2)
       order_3.item_orders.create!(item: @tire, price: @tire.price, quantity: 2)
 
-      expect(@meg.distinct_cities).to eq(["Denver","Hershey"])
+      expect(@meg.distinct_cities).to include("Hershey")
+      expect(@meg.distinct_cities).to include("Denver")
     end
```

Comment:
https://makandracards.com/makandra/899-check-if-two-arrays-contain-the-same-elements-in-ruby-rspec-or-test-unit